### PR TITLE
[Validator] Add missing Slovak translations.

### PR DESCRIPTION
**Pull request by miromichalicka**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no 
| Tickets       | #38762
| License       | MIT

This PR adds missing Slovak Validator translations

### CODE REVIEW

**Review by Nyholm — APPROVED** [review]
Thank you

**Comment by miromichalicka** [comment]
The fabbot.io error is false positive, there really shouldn't be double `s` in the word.

**Comment by Nyholm** [comment]
Correct. Fabbot does not handle Slovak really well =)

File: src/Symfony/Component/Validator/Resources/translations/validators.sk.xlf [hunk 1]
```diff
@@ -366,6 +366,26 @@
...
+                <source>This value is not a valid hostname.</source>
+                <target>Táto hodnota nie je platný hostname.</target>
+            </trans-unit>
+            <trans-unit id="96">
+                <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
+                <target>Počet prvkov v tejto kolekcii musí byť násobok {{ compared_value }}.</target>
+            </trans-unit>
+            <trans-unit id="97">
+                <source>This value should satisfy at least one of the following constraints:</source>
+                <target>Táto hodnota musí spĺňať aspoň jedno z nasledovných obmedzení:</target>
```
Comment (signor-pedro) [inline src/Symfony/Component/Validator/Resources/translations/validators.sk.xlf]: Spravnejsie by asi bolo "nasledujúci" vid
https://jazykovaporadna.sme.sk/q/1581/

 https://slovnik.juls.savba.sk/?w=nasledovný&s=exact&c=ye29&cs=&d=kssj4&d=psp&d=sssj&d=orter&d=scs&d=sss&d=peciar&d=hssjV&d=bernolak&d=noundb&d=orient&d=locutio&d=obce&d=priezviska&d=un&d=pskcs&d=psken#

Ale toto uz je uplna blbost, kludne sa moze mergnut

(perhaps there is a better word, not important - can be merged as is)

**Comment by derrabus** [comment]
Thank you @miromichalicka.

**Pull request merged**